apis/lambda/v1alpha1/custom_types.go

```diff
@@ -43,11 +47,39 @@ type CustomFunctionParameters struct {
 	// +optional
 	RoleSelector *xpv1.Selector `json:"roleSelector,omitempty"`
 
-	CustomFunctionVPCConfigParameters CustomFunctionVPCConfigParameters `json:"vpcConfig,omitempty"`
+	// For network connectivity to AWS resources in a VPC, specify a list of security
+	// groups and subnets in the VPC. When you connect a function to a VPC, it can
+	// only access resources and the internet through that VPC. For more information,
+	// see VPC Settings (https://docs.aws.amazon.com/lambda/latest/dg/configuration-vpc.html).
+	CustomFunctionVPCConfigParameters *CustomFunctionVPCConfigParameters `json:"vpcConfig,omitempty"`
+
+	// The code for the function.
+	// +kubebuilder:validation:Required
+	CustomFunctionCodeParameters *CustomFunctionCodeParameters `json:"code"`
+}
+
+// CustomFunctionCodeParameters includes custom fields for FunctionCode struct.
+type CustomFunctionCodeParameters struct {
```

Comment:
@smcavallo That was the only way I could get `Function` to see the references. Is there something I'm missing that made it work the last time?